content/id/includes/index.md

```diff
@@ -0,0 +1,3 @@
+---
+headless: true
```

Comment:
what is this `index.md` file for?

Reply:
Hmm not sure but in the `en` they have this